mne/io/brainvision/brainvision.py

```diff
@@ -463,7 +463,7 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
 
     # Try to get measurement date from marker file
     # Usually saved with a marker "New Segment", see BrainVision documentation
-    regexp = r'^Mk\d+=New Segment,.*,\d+,\d+,\d+,(\d{20})$'
+    regexp = r'^Mk\d+=New Segment,.*,\d+,\d+,[-]?\d+,(\d{20})$'
```

Comment:
```suggestion
    regexp = r'^Mk\d+=New Segment,.*,\d+,\d+,-?\d+,(\d{20})$'
```
IIUC you should not need `[-]`, `-` should suffice (`?` should apply to the character immediately preceding it)